front/pages/w/[wId]/members/index.tsx

```diff
@@ -879,9 +879,9 @@ function ChangeMemberModal({
         await handleMembersRoleChange({
           members: [member],
           role: selectedRole,
-          mutate,
           sendNotification,
         });
+        await mutate(`/api/w/${member.workspaces[0].sId}/members`);
```

Comment:
Can we use the workspace here instead of relying on members?

Reply:
would require adding an arg to the function, do you feel it's justified?